db/migrate/20250120151959_add_unique_event_index_on_fees.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddUniqueEventIndexOnFees < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :fees, %i[pay_in_advance_event_transaction_id charge_id charge_filter_id], unique: true,
+      where: "created_at > '#{Time.current}'", algorithm: :concurrently
```

Comment:
Why the added created at? NULL values don’t matter I think 🤔

Reply:
@nudded We have duplicated values in the database, without this condition, the index will fail to create